reliability-engineering/pipelines/concourse-deployer.yml

```diff
@@ -241,38 +243,48 @@ jobs:
       var_files:
       - concourse-provider-config/version.tfvars
 
-- name: configure-teams
+- name: generate-roll-team-workers
   serial: true
   plan:
-  - in_parallel:
-    - get: tech-ops-private
-      passed: [scale-in]
-      trigger: true
-    - get: tech-ops
-      attempts: 10
-      passed: [scale-in]
-      trigger: true
-    - get: concourse-release
-      passed: [scale-in]
-      trigger: true
+  - get: tech-ops
+    attempts: 10
+    passed: [update]
+    trigger: true
   - task: generate-roll-instances-pipeline
     file: tech-ops/reliability-engineering/pipelines/tasks/generate-roll-instances-pipeline.yml
     params:
       DEPLOYMENT_NAME: ((deployment_name))
-      CONCOURSE_USERNAME: main
-      CONCOURSE_PASSWORD: ((readonly_local_user_password))
+      FLY_USERNAME: main
+      FLY_PASSWORD: ((readonly_local_user_password))
   - set_pipeline: roll-instances
     file: roll-instances-pipeline/roll-instances.json
+    vars:
+      deployment_branch: ((deployment_branch))
+      deployment_tag: ((deployment_tag))
+      deployment_name: ((deployment_name))
+
+- name: configure-info-pipeline
+  serial: true
+  plan:
+  - in_parallel:
+    - get: tech-ops-private
+      attempts: 10
+      passed: [update]
+      trigger: true
+    - get: tech-ops
+      attempts: 10
+      passed: [update]
+      trigger: true
   - task: generate-deploy-info-pipelines-pipeline
     file: tech-ops/reliability-engineering/pipelines/tasks/generate-deploy-info-pipelines-pipeline.yml
     params:
       DEPLOYMENT_NAME: ((deployment_name))
-      CONCOURSE_USERNAME: main
-      CONCOURSE_PASSWORD: ((readonly_local_user_password))
+      FLY_USERNAME: main
+      FLY_PASSWORD: ((readonly_local_user_password))
   - set_pipeline: deploy-info-pipelines
     file: deploy-info-pipelines-pipeline/deploy-info-pipelines.json
     vars:
-      deployment_branch: ((deployment_branch))
+      deployment_branch: master
```

Comment:
why are we changing this?

Reply:
I think this the remnants of a hack that we put in to work off-branch, because the `((deployment_branch))` mechanism expects an identically named branch in both `tech-ops` and `tech-ops-private` which we don't have.

Should be removed.

Reply:
Mistake, will fix that.